docs/getting-started/docker.md

```diff
@@ -14,48 +14,18 @@ Using Docker to build software allows you to run and test code without having to
 
 You will need to install [Docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) before following the instructions below.
 
-> **Note**
->
-> Our configuration uses [docker-compose.override.yml](https://docs.docker.com/compose/extends/#understanding-multiple-compose-files) which exposes Saleor, PostgreSQL, and Redis ports and runs Saleor via `python manage.py runserver` for local development. 
-
-If you do not wish to use any overrides, you can order compose to only use docker-compose.yml configuration using `-f`, for example: `docker-compose -f docker-compose.yml up`.
-
 
 ## Using local assets
 
 By default, we do not mount assets for development in the Docker. This is because the assets are built in the Docker at build-time and are not present in the cloned repository. As a result, what has been built on the Docker, would be subsequently overshadowed by empty directories from the host.
 
-However, we are aware that in some instances you may wish to mount the assets and see your changes reflected in the container. If so, modify the docker-compose.override.yml before proceeding.
+However, we are aware that in some instances you may wish to mount the assets and see your changes reflected in the container. If so, modify the docker-compose.yml before proceeding.
```

Comment:
Wouldn't you want to make a `docker-compose.override.yml` instead? Seems like a better practice, but maybe harder for new comers to Docker

Reply:
You could link that documentation link back here as a note: https://docs.docker.com/compose/extends/#understanding-multiple-compose-files

And add the command to run it:
```
docker-compose -f docker-compose.yaml -f docker-compose.override.yaml up
```

Reply:
@NyanKiyoshi command suggested can be simplified to `docker-compose up` - if these files are named as you suggested, compose will take it all into consideration.

The reason behind moving only compose file here (merge Saleors compose and override files + added some extra services) and dropping the override is that we use compose for only one environment - development.

Since there is no other env for compose I don't see a need for override file, furthermore with override added it can be confusing for new Docker new comers (which might not know that override is magically taken into consideration).